tools/webhook-apicoverage/resourcetree/README.md

```diff
@@ -0,0 +1,6 @@
+# Resource Tree
+
+resourcetree package contains types and interfaces that defines a resource tree(n-ary tree based representation of an API resource). Each resource tree is composed of nodes which have data encapsulated inside [nodeData](node.go) and operations that can be performed on each node in the interface [INode](node.go). Type of a node is logically defined by reflect.Kind. Each node type is expected to satisfy the [INode](node.go) interface.  
+
+##Resource Forest  
+[ResourceForest](resourceforest.go) groups all resource trees that are part an API version into a single construct and defines operations that span across them. As an example for Knative Serving, we will have individual resource trees for Configuration, Revision, Route and Service which are encapsulated inside a resource forest under version v1alpha1. Example of an operation that spans resource trees would be to get coverage details for outlined types connected using ConnectedNodes. ConnectedNodes are connections between nodes that are of same type(reflect.Type) and belong to same package but span across different trees.
```

Comment:
part-> part of
Service Which are -> Service, and they are
Example of an operation -> An example operation
ConnectedNodes are -> ConnectedNodes represent
"outlined types":  need more explanations for it. Or give an example. 
Also would be good to give an example of ConnectedNodes.

Reply:
Fixed and added an example.